client/src/components/cv/EditCV.tsx

```diff
@@ -215,29 +274,65 @@ function EditCV(props: Props) {
     }
   };
 
-  const getDataFromRefs = (refs: RefObject<FormInstance>[]) => {
-    const formsHaveErrors = refs.some(ref => {
+  const checkFormsForValidationErrors = () => {
+    const hasErrors = currentRefs.some(ref => {
```

Comment:
Please don't do it so complicated you don't need an array here. Better create some function like
```javascript
const hasInvalidFields = (form: SomeType | null) => !form
  ? false
  : form.getFieldsError().some(field => field.errors.length > 0)
```

and then use it like:
```javascript
hasInvalidFields(userFormRef.current) || hasInvalidFields(contactsFormRef.current)
```

Reply:
done